src/QMCWaveFunctions/Jastrow/CountingJastrowOrbital.h

```diff
@@ -0,0 +1,742 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2016 Jeongnim Kim and QMCPACK developers.
+//
+// File developed by: Brett Van Der Goetz, bvdg@berkeley.edu, University of California at Berkeley
+//
+// File created by: Brett Van Der Goetz, bvdg@berkeley.edu, University of California at Berkeley
+//////////////////////////////////////////////////////////////////////////////////////
+
+#ifndef QMC_PLUS_PLUS_COUNTING_JASTROW_ORBITAL_H
+#define QMC_PLUS_PLUS_COUNTING_JASTROW_ORBITAL_H
+
+#include "Configuration.h"
+#include "Particle/ParticleSet.h"
+#include "QMCWaveFunctions/WaveFunctionComponent.h"
+#include "QMCWaveFunctions/Jastrow/CountingRegion.h"
+
+namespace qmcplusplus
+{
+
+template <class RegionType>
+class CountingJastrowOrbital: public WaveFunctionComponent 
+{
```

Comment:
Could you change the class and file names to CountingJastrow. Calling it Orbital only adds confusion.

Reply:
I changed it.